cukinia

```diff
@@ -235,6 +242,10 @@ cukinia_runner()
 {
 	local ret
 
+    if [ ! -z "$__step" ]; then
```

Comment:
`if [ ! -z "$string ]` equals to `if [ "$string" ] which I would use for easier read-iness